beacon-chain/node/node.go

```diff
@@ -308,6 +308,16 @@ func (b *BeaconNode) registerP2P(cliCtx *cli.Context) error {
 			bootnodeAddrs[i] = string(b)
 		}
 	}
+	if bootstrapFile := cliCtx.String(cmd.BootStrapNodeFile.Name); bootstrapFile != "" {
```

Comment:
Can we refactor this into its own function and add a test to it?

Reply:
Yes, I can work on refactoring it.  I can work on adding a test to node_test as well.